.drone.star

```diff
@@ -838,29 +862,22 @@ def yarnlint(ctx):
 
     return pipelines
 
-def checkForRecentBuilds(ctx):
+def cancelPreviousBuilds():
     return [{
         "kind": "pipeline",
         "type": "docker",
-        "name": "stop-recent-builds",
-        "workspace": {
-            "base": dir["base"],
-            "path": config["app"],
+        "name": "cancel-previous-builds",
+        "clone": {
+            "disable": True,
```

Comment:
What are we introducing here?

Reply:
To check and cancel the previous running build, the pipeline doesn't need to clone the actual code

Reply:
Okay, thanks for clarifying!:)